aws/ig/main.tf

```diff
@@ -1,9 +1,40 @@
+locals {
+  ig_spec = {
+    apiVersion = "kops/v1alpha2"
+    kind = "InstanceGroup"
+    metadata = {
+      labels = {
+        "kops.k8s.io/cluster" = var.cluster-name
+      }
+      name = var.name
+    } 
+    spec = merge({
+      cloudLabels = var.cloud-labels
+      nodeLabels = var.node-labels
+      associatePublicIp = var.visibility == "public"
+      image = var.image
+      machineType = var.type
+      maxSize = var.max-size
+      minSize = var.min-size
+      role = "Node"
```

Comment:
Are we no longer using this as a variable anymore?

Reply:
We never did - check the removed `aws/ig/ig-spec-template.tf` above :)

Reply:
Perhaps he meant the wrong one.